ballerina/tests/simple-params-query-test.bal

```diff
@@ -780,6 +780,124 @@ function queryArrayBasicNullParams() returns error? {
     }
 }
 
+@test:Config {
+    groups: ["query", "query-simple-params"]
+}
+function testInOperator1() returns error? {
+    int[] ids = [1,2,3];
+    ParameterizedQuery sqlQuery = queryConcat(`SELECT count(*) as total FROM DataTable`,
+                                              ` WHERE row_id in (${ids[0]}, ${ids[1]}, ${ids[2]})`);
```

Comment:
```suggestion
    ParameterizedQuery sqlQuery = queryConcat(`SELECT count(*) as total FROM DataTable WHERE row_id in (`,
                                              `${ids[0]}, ${ids[1]}, ${ids[2]})`);
```

Reply:
Added new testcase for this. https://github.com/ballerina-platform/module-ballerina-sql/pull/367/files#diff-e2e5a38967208c0c5bd499529715b43f511030da68452e7e70dabcd43cd42a62R916